polling/processor.py

```diff
@@ -0,0 +1,59 @@
+"""
+Simplified polling processor which gets dispatches then waits to see if they're in progress
+
+Precursor to the more sophisticated poller.py
+"""
+
+import datetime
+import os
+import time
+
+import requests
+import urllib
+from dateutil import parser as date_parser
+
+
+VOLTUS_API_URL = os.getenv("VOLTUS_API_URL", "https://api.voltus.co")
+VOLTUS_API_KEY = os.getenv("VOLTUS_API_KEY", "secret")
+
+processed_dispatches = set()
+
+s = requests.Session()
+s.headers.update({"X-Voltus-API-Key": VOLTUS_API_KEY, "Accept": "application/json"})
+
+url = urllib.parse.urljoin(VOLTUS_API_URL, "/2022-04-15/dispatches")
+resp = s.get(url)
+dispatches = resp.json()["dispatches"]
+
+
+if __name__ == "__main__":
+    while True:
+        for dispatch_info in dispatches:
+            if dispatch_info["id"] in processed_dispatches:
+                continue
+            start_time = date_parser.parse(dispatch_info["start_time"])
+            end_time = (
+                date_parser.parse(dispatch_info["end_time"])
+                if dispatch_info.get("end_time")
+                else None
+            )
+            now = datetime.datetime.now(datetime.timezone.utc)
+            if start_time <= now:
+                if end_time is None or end_time >= now:
+                    processed_dispatches.add(dispatch_info["id"])
+                    print("Dispatch {} is in progress".format(dispatch_info["id"]))
+                    if dispatch_info["authorized"] and not dispatch_info.get(
+                        "cancelled"
+                    ):
```

Comment:
"Cancelled" isn't an attribute in dispatch_info

Reply:
Didn't we talk about this already? The docs say this field could be present: https://api.voltus.co/2022-04-15/docs/#list-recent-dispatches

Reply:
Ah got it, I filed a PR for that: https://github.com/voltusdev/voltus/pull/22664